spec/vorpal/unit/config_builder_spec.rb

```diff
@@ -43,7 +43,7 @@ class Tester; end
     end
 
     it 'does not redefine constants' do
-      stub_const('A::B::C::TestDB', 1)
```

Comment:
And `undefine_test_db_class` must be in the `after` block because we want all the tests to clean up after themselves, I see. This smells like the test is too coupled to the implementation here. A hint for some further refactoring.